### DISCUSSION TITLE
我的朋友们 - ek1ng's Blog

### DISCUSSION THREAD

**Discussion by giscus** [ek1ng/blog#27] [Announcements]
# 我的朋友们 - ek1ng's Blog

foucs on security and web development - ek1ng - ek1ng's Blog

https://ek1ng.com/friends/

**Comment by ek1ng** [comment]
欢迎在评论区留下ID和站点互换友链哇

**Reply by ChrisL0tus** [reply]
e哥e哥加我加我(比心
ID:L0tus
站点:https://l0tus.vip

**Reply by ek1ng** [reply]
已添加，学弟加油哇🔥

**Comment by pysnow1** [comment]
师傅，换个友链呗
ID：pysnow
站点：https://pysnow.cn

**Reply by ek1ng** [reply]
欢迎师傅多多交流，友链已添加

**Comment by GoldenSheep402** [comment]
学长，来换个友链
ID: GoldenSheep
站点: https://glodensheep.top/

**Reply by ek1ng** [reply]
添加啦
